control_plane/src/endpoint.rs

```diff
@@ -880,10 +882,13 @@ impl Endpoint {
                 self.external_http_address.port()
             ))
             .header(CONTENT_TYPE.as_str(), "application/json")
-            .body(format!(
-                "{{\"spec\":{}}}",
-                serde_json::to_string_pretty(&spec)?
-            ))
+            .body(
+                serde_json::to_string(&ConfigurationRequest {
+                    spec,
+                    compute_ctl_config: ComputeCtlConfig::default(),
```

Comment:
Can you remove it here to check that the default really works? 🙃

I worry about backward compatibility with current cplane

Reply:
The control plane always sends it. See https://github.com/neondatabase/neon/pull/10746#issuecomment-2654261333

Reply:
Yes, but i) it's not yet released; ii) it could be rolled back after release. So instead of making a lock/assumption that cplane is fully released and won't be rolled back, we could just make sure that compute works well if it's not passed. Or do I miss something?

Reply:
FYI, [this](https://github.com/neondatabase/cloud/pull/24059) was released yesterday https://neondb.slack.com/archives/C06SH3LARTP/p1739297612266619